pkg/cli/admin/release/release.go

```diff
@@ -20,9 +24,25 @@ func NewCmd(f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Comm
 			registries.
 			`),
 	}
+	cmd.PersistentFlags().StringVar(&GraphArch, "graph-arch", "", "Specify the architecture to find in the release graph (amd64|arm64|ppc64le|s390x) when not connected to a cluster")
 	cmd.AddCommand(NewInfo(f, streams))
 	cmd.AddCommand(NewRelease(f, streams))
 	cmd.AddCommand(NewExtract(f, streams))
 	cmd.AddCommand(NewMirror(f, streams))
 	return cmd
 }
+
+//Check to ensure that --graph-arch is a valid input
+func checkReleaseArch() error {
+	switch GraphArch {
+	case "x86_64":
+		GraphArch = "amd64"
```

Comment:
we could do the mapping in place when building the request and just check for invalid formats here

Reply:
Hey @atiratree What do you mean by "mapping in place" I didn't see a way to rename/remap values passed in flags? I did pull remapping out of the case logic, but I don't think that is what you meant?

Reply:
I meant resolving the special cases (x86_64, aarch64) in the replaceStableSemanticArgs function, but It is matter of preference - we can leave it here as well